zqd/space/filespace.go

```diff
@@ -0,0 +1,84 @@
+package space
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/brimsec/zq/zqd/api"
+	"github.com/brimsec/zq/zqe"
+)
+
+const PcapIndexFile = "packets.idx.json"
```

Comment:
This can be made private now.

Reply:
turns out it's still referenced by a test in handlers_zeek_test.go